src/engraving/libmscore/property.h

```diff
@@ -156,6 +156,8 @@ enum class Pid {
     SPACE,            // used for spacer
     TEMPO,
     TEMPO_FOLLOW_TEXT,
+    TEMPO_EQUATION,
+    TEMPO_EQUATION_VISIBLE,
```

Comment:
I'd be concerned about inserting new values into this enum, in case there's any code anywhere that relies on the existing values (esp. with serialization). Adding at the end is always safer.